dependencyManagement/build.gradle.kts

```diff
@@ -7,7 +7,7 @@ data class DependencySet(val group: String, val version: String, val modules: Li
 val dependencyVersions = hashMapOf<String, String>()
 rootProject.extra["versions"] = dependencyVersions
 
-val otelInstrumentationVersion = "2.8.0-alpha"
+val otelInstrumentationVersion = "2.9.0-alpha-SNAPSHOT"
```

Comment:
The commit was merged just after 2.8.0 release, so we will have a snapshot dependency until 2.9.0 is released.